calibrators/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,123 @@
+| `tag_sizes`                                   | `std::vector<double>` | `[0.6, 0.6, 0.6, 0.6, 0.6, 0.6]` | Payload sizes of the tags corresponding to the IDs in meters. The size of the vector should be the same as the size of `tag_ids`.                                                                                                                                       |
+| `lidartag_max_convergence_translation`        | `double`              | `0.05`                           | Threshold in meters for the translation component of the Kalman filter's covariance matrix to consider the hypothesis as converged.                                                                                                                                     |
+| `lidartag_max_convergence_translation_dot`    | `double`              | `0.03`                           | Threshold in meters per second for the velocity component of the Kalman filter's covariance matrix to consider the hypothesis as converged.                                                                                                                             |
+| `lidartag_max_convergence_rotation`           | `double`              | `3.0`                            | Threshold in degrees for the rotation component of the Kalman filter's covariance matrix to consider the hypothesis as converged.                                                                                                                                       |
+| `lidartag_max_convergence_rotation_dot`       | `double`              | `2.5`                            | Threshold in degrees per second for the angular velocity component of the Kalman filter's covariance matrix to consider the hypothesis as converged.                                                                                                                    |
+| `lidartag_new_hypothesis_translation`         | `double`              | `0.1`                            | Translation threshold in meters for generating a new hypothesis in lidartag tracking.                                                                                                                                                                                   |
+| `lidartag_new_hypothesis_rotation`            | `double`              | `15.0`                           | Rotation threshold in degree for generating a new hypothesis in lidartag tracking.                                                                                                                                                                                      |
```

Comment:
degrees

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/0de1d6da79fdfda706df17b8f4b5985ca7d9b82d